src/category_theory/category/Pointed.lean

```diff
@@ -79,6 +86,49 @@ between them. -/
   hom_inv_id' := Pointed.hom.ext _ _ e.symm_comp_self,
   inv_hom_id' := Pointed.hom.ext _ _ e.self_comp_symm }
 
+instance monoidal_category : monoidal_category Pointed.{u} :=
+{ tensor_obj := λ X Y, ⟨X × Y, (X.point, Y.point)⟩,
+  tensor_hom := λ W X Y Z f g, ⟨prod.map f.to_fun g.to_fun, prod.ext f.map_point g.map_point⟩,
+  tensor_unit := ⟨punit, punit.star⟩,
+  associator := λ X Y Z, iso.mk (equiv.prod_assoc _ _ _) rfl,
+  left_unitor := λ X, iso.mk (equiv.punit_prod _) rfl,
+  right_unitor := λ X, iso.mk (equiv.prod_punit _) rfl }
+
+instance : symmetric_category Pointed.{u} := { braiding := λ X Y, iso.mk (equiv.prod_comm _ _) rfl }
+
+@[simp] lemma coe_tensor (X Y : Pointed.{u}) : ↥(X ⊗ Y) = (X × Y : Type u) := rfl
+@[simp] lemma point_tensor (X Y : Pointed) : (X ⊗ Y).point = (X.point, Y.point) := rfl
+
+@[simp] lemma coe_unit : ↥(𝟙_ Pointed) = punit := rfl
+@[simp] lemma point_unit : (𝟙_ Pointed).point = () := rfl
+
+namespace monoidal_category
+variables {W X Y Z : Pointed.{u}}
+
+@[simp] lemma tensor_apply (f : W ⟶ X) (g : Y ⟶ Z) (p : W ⊗ Y) : (f ⊗ g) p = (f p.1, g p.2) :=
+rfl
+
+@[simp] lemma left_unitor_hom_apply {x : X} {p : punit} :
+  ((λ_ X).hom : (𝟙_ (Pointed.{u})) ⊗ X → X) (p, x) = x := rfl
+@[simp] lemma left_unitor_inv_apply {x : X} :
+  ((λ_ X).inv : X ⟶ (𝟙_ (Pointed.{u})) ⊗ X) x = (punit.star, x) := rfl
+
+@[simp] lemma right_unitor_hom_apply {x : X} {p : punit} :
+  ((ρ_ X).hom : X ⊗ (𝟙_ (Pointed.{u})) → X) (x, p) = x := rfl
+@[simp] lemma right_unitor_inv_apply {x : X} :
+  ((ρ_ X).inv : X ⟶ X ⊗ (𝟙_ (Pointed.{u}))) x = (x, punit.star) := rfl
+
+@[simp] lemma associator_hom_apply {x : X} {y : Y} {z : Z} :
+  ((α_ X Y Z).hom : (X ⊗ Y) ⊗ Z → X ⊗ (Y ⊗ Z)) ((x, y), z) = (x, (y, z)) := rfl
+@[simp] lemma associator_inv_apply {x : X} {y : Y} {z : Z} :
+  ((α_ X Y Z).inv : X ⊗ (Y ⊗ Z) → (X ⊗ Y) ⊗ Z) (x, (y, z)) = ((x, y), z) := rfl
+
+@[simp] lemma braiding_hom_apply {x : X} {y : Y} :
+  ((β_ X Y).hom : X ⊗ Y → Y ⊗ X) (x, y) = (y, x) := rfl
+@[simp] lemma braiding_inv_apply {x : X} {y : Y} :
+  ((β_ X Y).inv : Y ⊗ X → X ⊗ Y) (y, x) = (x, y) := rfl
+
+end monoidal_category
```

Comment:
Perhaps you'd also like to have that the forgetful functor to Type is monoidal? More than happy for this to be a followup PR though